[2021-08-12 15:36:14,563] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag_03.task_5 2021-08-12T13:29:08.614696+00:00 [queued]>
[2021-08-12 15:36:14,582] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag_03.task_5 2021-08-12T13:29:08.614696+00:00 [queued]>
[2021-08-12 15:36:14,583] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 15:36:14,583] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-12 15:36:14,583] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 15:36:14,605] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_5> on 2021-08-12T13:29:08.614696+00:00
[2021-08-12 15:36:14,609] {standard_task_runner.py:52} INFO - Started process 12028 to run task
[2021-08-12 15:36:14,623] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag_03', 'task_5', '2021-08-12T13:29:08.614696+00:00', '--job-id', '145', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task_example.py', '--cfg-path', '/tmp/tmp67vx67vy', '--error-file', '/tmp/tmpul3kmdjh']
[2021-08-12 15:36:14,626] {standard_task_runner.py:77} INFO - Job 145: Subtask task_5
[2021-08-12 15:36:14,670] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag_03.task_5 2021-08-12T13:29:08.614696+00:00 [running]> on host adam
[2021-08-12 15:36:14,745] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag_03
AIRFLOW_CTX_TASK_ID=task_5
AIRFLOW_CTX_EXECUTION_DATE=2021-08-12T13:29:08.614696+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-12T13:29:08.614696+00:00
[2021-08-12 15:36:14,746] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-12 15:36:14,747] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo hello from airflow > ~/Documents/hello_airflow.txt']
[2021-08-12 15:36:14,767] {subprocess.py:74} INFO - Output:
[2021-08-12 15:36:14,769] {subprocess.py:82} INFO - Command exited with return code 0
[2021-08-12 15:36:14,806] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=parallel_dag_03, task_id=task_5, execution_date=20210812T132908, start_date=20210812T133614, end_date=20210812T133614
[2021-08-12 15:36:14,836] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-12 15:36:14,846] {local_task_job.py:149} INFO - Task exited with return code 0
